### ISSUE TITLE
Make the distinction between the role types more clear

### ISSUE THREAD

**Issue by kajdehoop** [xrnl/aginun#93]
## Problem that this feature solves
Right now the only clear distinction between different types of roles is the background color of their respective cards, but some types like "Coordination Circle" and "Outreach & Training" have the same color so it looks as if they belong to the same group. 

## Feature description
We can add more colors, but this can make the app look messy. It may be better to use a different icon for each type of role.

## Additional comments
Maybe we can also highlight the most important (or urgent) roles in some way. But we should be careful to prevent people from always adding an 'urgent' tag (or something similar) to a role to be displayed on top of the list.

**Comment by SamLubbers** [comment]
@kajdehoop good that you spotted this issue. It's something that fresh eyes easily see but that I don't see after looking at the application for so long haha. 

The colours were implemented in #77. As you can see in the pull request, we used colours from the XR style guide. Since there are more working groups than colours, we need to reuse some colours, but as you mentioned these colours are not entirely logical right now. We should actually use similar colours for similar circles. For example, we could use the same colour for `Outreach & Training` and for `Integration` as those circles do similar work.

**Comment by kajdehoop** [comment]
Cool :) If you can make a list of the circles that are similar, I can set the colours in the styles store

**Comment by SamLubbers** [comment]
Here are all the circles of XR NL, grouped by similarity:

```
Outreach & Training
Integration

Action & Logistics
Rebellion Support Circle

Finance
Fundraising

Regenerative Culture
Inclusion & Power

Political Strategy & Change
Future of Democracy

Legal

Media & Communications

Arts in Action

Tech

Coordination Circle
```
You can adjust the colours for these in `store/modules/styles.js`

**Comment by SamLubbers** [comment]
This will be fixed with the new design of #167 Thanks Kaj!!
